pyproject.toml

```diff
@@ -38,6 +38,7 @@ python = ">=3.8.1,<4.0"
 click = "^8.1.3"
 pddl = "^0.3.1"
 pylogics = "^0.2.1"
+FDgrounder = { git = "https://github.com/LBonassi95/downward/", branch = "only-grounder-lib" }
```

Comment:
so, we need it? 👀

Reply:
@LBonassi95

Reply:
I see it used in `utils.pddl2ltl`:

```
from FDgrounder.pddl_parser.lisp_parser import parse_nested_list
```

and: in `utils.to_pddl`:
```
FDgrounder.pddl
```

Reply:
I remember @LBonassi95 saying we are not dependent on that anymore. Is that true?